a-few-things-you-should-know-about-javascript.md

```diff
@@ -45,15 +45,15 @@ Don’t waste time worrying about managing dependencies or other distractions, t
 
 ### If You Do Nothing Else, Be Consistent
 
-Use a style guide, have identifiable patterns. Having the same style and patterns means new projects will feel familiar. One of the biggest productivity killers is rummaging around in unfamiliar code.
+One of the biggest productivity killers is rummaging around in unfamiliar code. Use a style guide. Have identifiable patterns. The same style and patterns means new projects will feel familiar. **(When you make a point, state the shitty thing FIRST, then the good thing you're suggesting. That way you leave the reader with a positive impression.)**
```

Comment:
i was going to do this already, another friend made same suggestion, thanks for all of this!!